static/app/views/issueDetails/streamline/solutionsSection.tsx

```diff
@@ -74,48 +70,59 @@ export default function SolutionsSection({
     isPending: isSummaryLoading,
     isError: isSummaryError,
   } = useGroupSummary(group.id, group.issueCategory);
+  const {data: autofixSetupData, isPending: isAutofixSetupLoading} = useAutofixSetup({
+    groupId: group.id,
+  });
+
+  const isSampleError = useIsSampleEvent();
 
   const issueTypeConfig = getConfigForIssueType(group, group.project);
-  const hasSummary = isSummaryEnabled(
-    hasGenAIConsent,
-    issueTypeConfig.issueSummary.enabled,
-    organization.hideAiFeatures
-  );
-  const aiNeedsSetup =
-    !hasGenAIConsent &&
-    issueTypeConfig.issueSummary.enabled &&
-    !organization.hideAiFeatures;
+
+  const areAiFeaturesAllowed =
+    !organization.hideAiFeatures &&
+    getRegionDataFromOrganization(organization)?.name !== 'de';
+
+  const isSummaryEnabled = issueTypeConfig.issueSummary.enabled;
+  const isAutofixEnabled = issueTypeConfig.autofix;
   const hasResources = issueTypeConfig.resources;
 
-  return (
-    <div>
-      <SidebarSectionTitle style={{marginTop: 0}}>
-        <HeaderContainer>
-          {t('Solutions Hub')}
-          {hasSummary && (
-            <StyledFeatureBadge
-              type="beta"
-              title={tct(
-                'This feature is in beta. Try it out and let us know your feedback at [email:autofix@sentry.io].',
-                {
-                  email: <a href="mailto:autofix@sentry.io" />,
-                }
-              )}
-            />
-          )}
-        </HeaderContainer>
-      </SidebarSectionTitle>
-      {hasSummary && (
-        <Summary>
-          <GroupSummary
-            data={summaryData ?? undefined}
-            isError={isSummaryError}
-            isPending={isSummaryLoading}
-            preview
-          />
-        </Summary>
-      )}
-      {aiNeedsSetup && (
+  const hasSummary = hasGenAIConsent && isSummaryEnabled && areAiFeaturesAllowed;
+  const hasAutofix = isAutofixEnabled && areAiFeaturesAllowed && !isSampleError;
+
+  const needsGenAIConsent =
+    !hasGenAIConsent && (isSummaryEnabled || isAutofixEnabled) && areAiFeaturesAllowed;
+
+  const needsAutofixSetup =
+    isAutofixEnabled &&
+    !isAutofixSetupLoading &&
+    (!autofixSetupData?.genAIConsent.ok || !autofixSetupData?.integration.ok) &&
+    areAiFeaturesAllowed;
+
+  const showCtaButton = needsGenAIConsent || hasAutofix || (hasSummary && hasResources);
+  const isButtonLoading = isAutofixSetupLoading;
+
+  const getButtonText = () => {
+    if (isAutofixSetupLoading) {
+      return <LoadingIndicator mini />;
```

Comment:
Can we do a placeholder here too

Reply:
wait what, this should've been changed

Reply:
must've accidentally cmd+z'd in between all the chaos this morning